exporter/opentelemetry-exporter-otlp-proto-grpc/src/opentelemetry/exporter/otlp/proto/grpc/exporter.py

```diff
@@ -81,12 +81,16 @@ def environ_to_compression(environ_key: str) -> Optional[Compression]:
     return _ENVIRON_TO_COMPRESSION[environ_value]
 
 
-def _translate_key_values(key: Text, value: Any) -> KeyValue:
+def _translate_key_values(
+    key: Text, value: Any, max_attr_value_length: Optional[int]
```

Comment:
```suggestion
    key: Text, value: Any, max_attr_value_length: Optional[int] = None
```

Reply:
So I wanted it to be optional so other places could explicitly pass `None` but still wanted to force users to always pass a value. This was to ensure that any OTLP exporter does not forget to include the length limit in some invocations.

Reply:
I'm not sure I understand the explanation. Did you mean you wanted it to be explicitly passed in to `_translate_key_values` when used internally, but when using the OTLP exporter, users are not forced to pass it in?

Reply:
Since this is a private function, I don't expect or would like to not account for users using this function. 

What I meant was that it is very easy for a contributor to call this function somewhere without forwarding/passing the limit value if we keep it as an optional paramater. Since this function can be called from so many different places, it's easy for a contributor to forget to pass the limit value and for reviewers to not catch that. Making it a required param would catch all such mistakes while still allowing any possible use cases where limit is not desired (by setting it to None).